docs/resources/server/datamaps/builtin.md

```diff
@@ -98,6 +100,32 @@ Example:
 }
 ```
 
+## `neoforge:oxidizables`
+Allows configuring oxidation stages, as a replacement for `WeatheringCopper#NEXT_BY_BLOCK` (which will be ignored in 1.21.2). This data map is also used to build a reverse deoxidation map (for scraping with an axe). It is located at `neoforge/data_maps/block/oxidizables.json` and its objects have the following structure:
```

Comment:
New line between heading and paragraph please, and between paragraph and code block as well.

Reply:
```suggestion

Allows configuring oxidation stages, as a replacement for `WeatheringCopper#NEXT_BY_BLOCK` (which will be ignored in 1.21.2). This data map is also used to build a reverse deoxidation map (for scraping with an axe). It is located at `neoforge/data_maps/block/oxidizables.json` and its objects have the following structure:
```